app/kubernetes/helpers/configurationHelper.js

```diff
@@ -38,6 +38,23 @@ class KubernetesConfigurationHelper {
     });
   }
 
+  static getApplicationConfigurations(applications, configurations) {
+    const configurationsUsed = configurations.filter((config) => {
+      const configApplications = KubernetesConfigurationHelper.getUsingApplications(config, applications);
+      return configApplications && configApplications.length !== 0;
```

Comment:
can we make sure that `getUsingApplications` will return an array, so we can remove the first condition?